.github/workflows/push_staging.yaml

```diff
@@ -0,0 +1,33 @@
+# this build is triggered when a new pull request has been merged
+# into the main branch
+
+name: openverse/deploy/staging
+
+on:
+  push:
+    branches:
+      - main
+      - ci/* # allow any ci/* branch the ability to deploy to staging without the need to merge the PR
+
+jobs:
+  push:
+    name: push
+    runs-on: ubuntu-latest
+    environment:
+      name: staging
+      url: https://search-staging.openverse.engineering
```

Comment:
```
      url: https://search-staging.openverse.engineering
```

These URLs might be better as GitHub Secrets, so we can change them without needing to modify the actions and go through the code review process.

Reply:
Unfortunately, this approach won't work. The secrets are encrypted and hidden from visualization, for example, if you take a look at the [actions](https://github.com/WordPress/openverse-frontend/runs/5045757213?check_suite_focus=true) of this PR and click in the `docker build` step, you'll see the secrets are hidden by default with asterisks:

```log
      "***.dkr.ecr.***.amazonaws.com/openverse/frontend:latest"
```

So if we use secrets for this, the environment will display asterisks.

Anyway, I would consider it worth the try, if the result is exactly what I'm saying, I can look for an alternative workaround, for example we can introduce variables in the job by default, something that I need to investigate.

Reply:
Oh that's right, I forgot these URLs are also only _presentational_, they show up in the actions UI but aren't used in the actual pipeline. 👍